audio/codec2/Portfile

```diff
@@ -22,13 +22,15 @@ long_description    Codec 2 is an open source speech codec designed for \
     designed for digital voice over HF radio.
 homepage            http://www.rowetel.com/codec2.html
 
-github.setup        drowe67 codec2 d6f93b52601f2c099ff98688eed22d680dec96fe
-version             20191101-[string range ${github.version} 0 7]
-checksums           rmd160  34473d6d1b7ee4a44a0ad0f3a3e3bca60000279e \
-                    sha256  f446882d6578da2f578a8528e1ae8ee946b5bac2a1a2122381cb7e1e6eb9d4ff \
-                    size    12246011
+github.setup        drowe67 codec2 0.9.2 v
+checksums           rmd160  c7858aa8e633dc8451514db6461194c6fef26319 \
+                    sha256  60fd5b7929f74111e6ef8d61cfe207b76fee86c3f6ed153a2d3e64209016d489 \
+                    size    12244703
 revision            0
 
+# 20191112: move from commit to version
```

Comment:
This comment is not necessary and can be removed. Also the extra blank line can be removed.